client/my-sites/email/titan-add-mailboxes/titan-mailbox-pricing-notice.jsx

```diff
@@ -23,6 +23,66 @@ const doesAdditionalPriceMatchStandardPrice = ( domain, titanMonthlyProduct ) =>
 	);
 };
 
+function getPriceMessage( purchaseCost, expiryDate, translate ) {
+	return purchaseCost.amount === 0
+		? translate(
+				'You are still in your Professional Email trial period, so this mailbox is free for the remainder of your trial.',
+				{
+					args: {
+						expiryDate: expiryDate,
+					},
+					components: {
+						strong: <strong />,
+					},
+					comment: '%(expiryDate)s is a localized date (e.g. February 17, 2021)',
+				}
+		  )
+		: translate(
+				'You can purchase new mailboxes at the prorated price of {{strong}}%(proratedPrice)s{{/strong}} per mailbox.',
+				{
+					args: {
+						proratedPrice: purchaseCost.text,
+					},
+					components: {
+						strong: <strong />,
+					},
+					comment:
+						'%(proratedPrice)s is a formatted price for an email subscription (e.g. $3.50, €3.75, or PLN 4.50)',
+				}
+		  );
+}
+
+function getPriceMessageExplanation( purchaseCost, renewalCost, translate ) {
```

Comment:
As above RE the function arguments.
```suggestion
function getPriceMessageExplanation( { purchaseCost, renewalCost, translate } ) {
```